packages/adblocker/test/matching.test.ts

```diff
@@ -29,37 +29,33 @@ use((chai, utils) => {
   utils.addMethod(
     chai.Assertion.prototype,
     'matchRequest',
-    function (this: any, req: Partial<Request>) {
+    function (this: Chai.ChaiStatic & { _obj: NetworkFilter }, req: Partial<Request>) {
       const filter = this._obj;
       const request = Request.fromRawDetails(req);
 
       new chai.Assertion(filter).not.to.be.null;
 
-      this.assert(
-        filter.match(request),
-        'expected #{this} to match #{exp}',
-        'expected #{this} to not match #{exp}',
```

Comment:
From the Chai's documentation, I don't see the third argument of `assert`. Maybe this API is outdated.

Reply:
https://www.chaijs.com/api/assert/